packages/stream_chat_flutter/lib/src/theme/stream_chat_theme.dart

```diff
@@ -61,6 +58,8 @@ class StreamChatThemeData {
     StreamGalleryHeaderThemeData? imageHeaderTheme,
     StreamGalleryFooterThemeData? imageFooterTheme,
     StreamMessageListViewThemeData? messageListViewTheme,
+    @Deprecated(
+        "Use 'StreamChatThemeData.voiceRecordingAttachmentTheme' instead")
     StreamVoiceRecordingThemeData? voiceRecordingTheme,
```

Comment:
Can we construct a `StreamVoiceRecordingAttachmentThemeData` based on the `StreamVoiceRecordingThemeData` if the old theme data is given instead of the new one?